tests/release/OWNERS

```diff
@@ -1,8 +1,14 @@
-approvers:
+reviewers:
 - davidmogar
 - scoheb
 - theflockers
 - happybhati
 - johnbieren
 - mmalina
-- kasemAlem
\ No newline at end of file
+- kasemAlem
+- jinqi7
+- cuipinghuo
+
+approvers:
```

Comment:
I really don't think we should restrict approving to just Scott and David

Reply:
that would be a really, really bad idea

Reply:
thanks for the review. Update.